tests/test_components/test_IO.py

```diff
@@ -21,6 +21,12 @@
 SIM_DIR = "tests/sims"
 
 
+@pytest.fixture
+def split_string(monkeypatch):
```

Comment:
nice :)

Reply:
I had something similar worked out in a backend test already :)